ports/stm32/boards/STM32H747I_DISCO/mpconfigboard.h

```diff
@@ -0,0 +1,304 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ * The MIT License (MIT)
+ * Copyright (c) 2025
+ */
+
+#define MICROPY_HW_BOARD_NAME       "STM32H747I-DISCO"
+#define MICROPY_HW_MCU_NAME         "STM32H747"
+#define MICROPY_HW_FLASH_FS_LABEL   "STM32H747I-DISCO"
```

Comment:
I think the board should just use the standard flash label and not define a custom one (following most other boards).